applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -34,9 +34,13 @@ static struct media_player *local_player;
 static ble_mcs_play_pause_cb play_pause_cb;
 #endif /* (CONFIG_BT_MCS) */
 
-#if (CONFIG_BT_MCC)
-static bool mcs_discover_called;
-#endif /* (CONFIG_BT_MCC) */
+static uint8_t mcp_mcs_disc_status[CONFIG_BT_MAX_CONN];
```

Comment:
I would assume we don't need two mcp_mcs_disc_status?
Since MCC discover is from headset to gateway, so headset only need to maintain one discover state?

Reply:
This is true for now, but for future-proofing this is added.

Reply:
When MCC supports multiple connections, we will need to keep a state per ACL connection, as we will need to do a discover per connection (e.g. if you have a PC and a Phone connected to the same headset). 

I think it is fine to already support here now, even if MCC does not yet support multiple connections.

Reply:
I created https://github.com/zephyrproject-rtos/zephyr/issues/55324 to add support for multiple connections in MCC